lib/ios/TopBarAppearancePresenter.m

```diff
@@ -32,18 +32,20 @@ - (void)setTransparent:(BOOL)transparent {
 }
 
 - (void)updateBackgroundAppearance {
-    if (self.transparent) {
-        [self.getAppearance configureWithTransparentBackground];
-        [self.getScrollEdgeAppearance configureWithTransparentBackground];
-    } else if (self.backgroundColor) {
-        [self.getAppearance setBackgroundColor:self.backgroundColor];
-        [self.getScrollEdgeAppearance setBackgroundColor:self.backgroundColor];
-    } else if (self.translucent) {
-        [self.getAppearance configureWithDefaultBackground];
-        [self.getScrollEdgeAppearance configureWithDefaultBackground];
-    } else {
+    if (@available(iOS 13.0, *)) {
         [self.getAppearance configureWithOpaqueBackground];
```

Comment:
I think it make more sense to apply this two lines only when `self.backgroundColor` is available, to line 43